Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7049.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.UITest;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve (AllMembers=true)]
+	[Issue (IssueTracker.Github, 7049, "Null reference exception on some Android devices - Xamarin.Forms.Platform.Android.PageRenderer.Xamarin.Forms.Platform.Android.IOrderedTraversalController.UpdateTraversalOrder", PlatformAffected.Android)]
+	public class Issue7049 : TestContentPage
+	{
+		private const string ContinueButton = "ContinueButton";
```

Comment:
Just one more thing, then it looks all good. As per the coding standard we don't use the `private` keyword

Reply:
Done